### PR TITLE
Start setup!

### LINKED ISSUE #92: add this back
Students are required to have prior experience with R programming (or other languages) and need to have taken the following data science courses (or have equivalent experience): 

140.711 Advanced Data Science I and 140.712 Advanced Data Science II
Instructor permission is required to enroll in the course.

### LINKED ISSUE #97: Set up config file for dates, terms, etc

### PR DESCRIPTION
**Pull request by avahoffman**
- Create config file
- Do some cleanup of `index`, `evaluation`, `code_of_conduct`, and `syllabus` for clarity/brevity
- Resolves #92 
- Resolves #97 

@carriewright11  I think I'll work on the schedule, etc. in another PR cause this is getting kinda big.

Note: something funky is happening with the URL checker, so I've disabled it for now

### CODE REVIEW

**Comment by avahoffman** [comment]
@carriewright11 I think this is ready for your review. It doesn't have everything we discussed, but I think it might be harder to review everything all at once

File: syllabus.Rmd [hunk 1]
```diff
@@ -1,64 +1,82 @@
...
 
 <!-- <a href="syllabus.pdf" target="_blank" download = "syllabus.pdf"  ><button class="btn"><i class="fas fa-file-alt"></i> PDF version</button></a> -->
 
-
+<br>
 
 ### Course Description
 
-This course is focused on helping students learn how to apply their data science knowledge in non-academic settings. Students will partner with community-based organizations (CBOs) in Baltimore to work on a data science related goal. Students will focus on developing critical service-learning and professional skills as they work with CBOs to accomplish a data science related goal.
+This course is focused on helping students learn how to apply their data science knowledge in non-academic settings. Students will partner with [community-based organizations (CBOs)](https://source.jhu.edu/about/partner-organizations) in Baltimore to work on a data science related goal. Students will focus on developing critical service-learning and professional skills as they work with CBOs to accomplish a data science related goal. _This course can feel uncomfortable: students are expected to be self motivated and work on solutions to problems without being told what to do._
```
Comment (carriewright11) [inline syllabus.Rmd:23]: maybe we should add something about how this gives a realistic experience of working in academia as a PI or working in industry or something?

File: index.Rmd [hunk 2]
```diff
@@ -4,52 +4,46 @@ output:
   html_document
 ---
 
-<div class = "banner">
-Welcome to the course website for Baltimore Community Data Science (PH140.840)!  
-</div>
-
-### **About**
-This course was developed in partnership with [SOURCE](https://source.jhu.edu/), the community engagement and service-learning center at the Johns Hopkins Bloomberg School of Public health, which is focused on community engagement that involves:
```
[the remark this answers is not in the document]
Comment (carriewright11) [reply index.Rmd:12]: nevermind I see this now in the preview... hard to tell I guess just looking at diffs when there are lots of changes :P

File: index.Rmd [hunk 3]
```diff
@@ -4,52 +4,46 @@ output:
   html_document
 ---
 
-<div class = "banner">
-Welcome to the course website for Baltimore Community Data Science (PH140.840)!  
-</div>
-
-### **About**
-This course was developed in partnership with [SOURCE](https://source.jhu.edu/), the community engagement and service-learning center at the Johns Hopkins Bloomberg School of Public health, which is focused on community engagement that involves:  
-**Social Change / Authentic Relationships / Redistributing Power**
```
[the remark this answers is not in the document]
Comment (carriewright11) [reply index.Rmd:13]: I see this too :)

**Comment by carriewright11** [comment]
I noticed that some bullets in the syllabus have lower case  and upper case (this is from before) - maybe it makes sense to fix that in this PR?

This looks awesome, @avahoffman!

**Comment by carriewright11** [comment]
> @carriewright11 I think this is ready for your review. It doesn't have everything we discussed, but I think it might be harder to review everything all at once

totally agree! thanks for doing it in bits

File: index.Rmd (hunk 2 above)
[the remark this answers is not in the document]
Comment (avahoffman) [reply index.Rmd:12]: yes, totally, hence breaking it up :)

**Pull request merged**